lsst-epo/investigations-client
# [F] add print to review page

JIRA: https://jira.lsstcorp.org/browse/EPO-8796

## What this change does ##

Add the print button to the review page along with print styles and a few general review page QOL changes

## Notes for reviewers ##

This is a clientside change only

Include an indication of how detailed a review you want on a 1-10 scale.
- 1 = "I barely need review on this"

## Testing ##

- Visit the review page while not logged in
- Observe that no name is filled out, input a name
- Click the print button, check that fields look correct and there are no undesired page breaks or other weirdness
- Check that the name entered shows on the downloaded filename
- Visit the review page while logged in
- Observe that a name is present, all else remains the same. 

## Checklist ##

If any of the following are true, please check them off

## Screenshots (optional) ##
### Before:
### After:
